runtime/lang/menu_da.latin1.vim

```diff
@@ -1,3 +1,3 @@
 " Menu Translations:	Danish for iso-8859-1 encoding
 
-source <sfile>:p:h/menu_da.utf-8.vim
+source <sfile>:p:h/menu_da.utf-8.vim
```

Comment:
Is this change needed?

Reply:
No i guess not.
The other files does have a newline at the end it seems.

Reply:
By other files i mean translations.